Code/C++/longest_common_subsequence.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+
+* longest common subsequence finds the longest subsequence that is present in both the strings.
+* A subsequence is defined as the sequence that is same relative order but not neccesarily in a contiguous manner.
+* Problem Statement : Given two strings s1 and s2 and we have to find the longest common subsequence from both the string
+* Approach used : In this problem a dynamic programming solution will be used as the recursive algorithm have a overlapping
+  subproblem which can be solved by using dynamic programming. We can use both tabulation and memoization but we will be using 
+  tabulation in bottom up manner.
+* First we will be considering a 2D array of size (n+1) * (m+1) where n and m are hthe size of first and second string respectively
+  and we are taking an extra column and row to get the value of the substring of length 1 and 1 we need to have to value of 01, 00, 10
+* Any position at i,j i.e dp[i][j] will denote the length of longest common subsequence of prefix string of s1[0...i-1] and prefix
+  string of s2[0...j-1]
+* An example to demostrate the table view of the solution is
```

Comment:
remove line from 13 to 16

Reply:
So I will remove the line from 13 to 28 and add it to the bottom of the code

Reply:
yes